src/base/bittorrent/sessionimpl.cpp

```diff
@@ -73,6 +73,9 @@
 #include <QThreadPool>
 #include <QTimer>
 #include <QUuid>
+#include <QNetworkAccessManager>
```

Comment:
`QNetworkAccessManager` isn't intended to be used directly in qBittorrent. `Net::DownloadManager` component should be used instead.